ci/Jenkinsfile

```
@@ -77,23 +77,26 @@ pipeline {
     stage('Finalizations') {
       stages {  /* parallel builds of minimal / mainnet not yet supported */
         stage('minimal') {
-          steps { script { timeout(26) {
-            launchLocalTestnet('minimal')
-          } } }
+          steps { timeout(26) {
+            sh 'make local-testnet-minimal'
+          } }
+          post { always {
+            sh 'tar cjf local-testnet-minimal.tar.gz local-testnet-minimal/*.txt'
+          } }
         }
 
         stage('mainnet') {
-          steps { script { timeout(62) {
-            launchLocalTestnet('mainnet')
-          } } }
+          steps { timeout(62) {
+            sh 'make local-testnet-mainnet'
+          } }
+          post { always {
+            sh 'tar cjf local-testnet-mainnet.tar.gz local-testnet-mainnet/*.txt'
+          } }
         }
       }
-    }
-
-    stage('Upload') {
-      steps { timeout(5) {
-        archiveArtifacts('*.tar.gz')
-      } }
+      post { always { timeout(5) {
```

Small nitpick but maybe:
```suggestion
      post {
        cleanup { timeout(5) {
```

---

Shouldn't matter unless multiple `post` actions are defined. `always` runs at the beginning of `post`, `cleanup` at the end. As the rest works using `always`, prob better to keep as is for now rather than finding out that the assumption is incorrect ^^

---

We already use `cleanup` in desktop. It works as expected.

---

Ah, great to know for next time (for the `cleanupWorkspace` action it already was on `always` before this PR btw)